### ISSUE TITLE
Question about eligible validators in rewards/penalties processing

### ISSUE THREAD

**Issue by NIC619** [ethereum/eth2.0-specs#857]
In spec v0.5.1, eligible validators are active validators in previous epoch:
```python
    # v0.5.1
    # Process rewards or penalties for all validators
    for index in get_active_validator_indices(state.validator_registry, get_previous_epoch(state)):
        ...
```

However they are changed to current epoch active validators in d1d1b73fb1783b564556a48ea86d69d8dd1003e7 but not mentioned in the commit message or PR description so I wonder if this is intended?
```python
    # dev
    eligible_validators = [
        index for index, validator in enumerate(state.validator_registry)
        if (
            is_active_validator(validator, current_epoch) or
            (validator.slashed and current_epoch < validator.withdrawable_epoch)
        )
    ]
    # Process rewards or penalties for all validators
    for index in eligible_validators:
```

And does it makes sense to change `all validators` in the comment `# Process rewards or penalties for all validators` to be something more specific?

**Comment by djrtwo** [comment]
You're right, this would be the previous epoch

**Comment by djrtwo** [comment]
This is being addressed in #949

**Comment by hwwhww** [comment]
closing in favor of #949
